src/libraries/System.Security.Cryptography/tests/X509Certificates/TestDataGenerator.cs

```diff
@@ -98,125 +99,22 @@ internal static void MakeTestChain4(
             using (RSA intermediateKey2 = RSA.Create())
             using (RSA endEntityKey = RSA.Create())
             {
-                ReadOnlySpan<RSA> keys = new[]
-                {
+                ReadOnlySpan<RSA> keys =
+                [
                     rootKey,
                     intermediateKey1,
-                    intermediateKey2,
+                    intermediateKey1,
```

Comment:
I assume you don't actually intend for the two intermediates to use the same key...?

Reply:
Ah yeah good catch. Doesn't matter  for this test but probably would if someone were trying to test things with SKIs.